integrationtests/src/test/java/org/axonframework/integrationtests/queryhandling/AbstractSubscriptionQueryTestSuite.java

```diff
@@ -89,36 +78,34 @@
  */
 public abstract class AbstractSubscriptionQueryTestSuite extends AbstractQueryTestSuite {
```

Comment:
Those tests still doesn't pass for the `DistributedQueryBusSubscriptionQueryTest` (this class is Disabled), so it should be under investigation. They were written for SimpleQueryBus so there might be some difference or they need to be adjusted to support both (eg. in async nature of the Distributed one?), but the TestSuite should eventually pass for both. 

I think the simplest test you may try to fix is: `emittingAnUpdate`.